code/game/objects/items/toys.dm

```diff
@@ -23,6 +23,8 @@
  *		Clockwork Watches
  *		Toy Daggers
  *		Eldrich stuff
+ * 		Yatzy
```

Comment:
?

Reply:
Old Yatzy PR. It was not included in the comments.